routing/result_interpretation.go

```diff
@@ -336,9 +336,9 @@ func (i *interpretedResult) processPaymentOutcomeIntermediate(
 		reportOutgoing()
 
 	// Some implementations use this error when the next hop is offline, so we
-	// do the same as FailUnknownNextPeer and process the channel update.
+	// do the same as FailUnknownNextPeer and also process the channel update.
 	case *lnwire.FailChannelDisabled:
-		
+
```

Comment:
Formatting changes can be squashed with previous commit?

Reply:
Will do